python/google/protobuf/json_format.py

```diff
@@ -119,7 +120,8 @@ def MessageToJson(
       use_integers_for_enums,
       descriptor_pool,
       float_precision,
-      including_default_value_without_presence_fields
+      including_default_value_without_presence_fields,
```

Comment:
Sorry about that, this file was touched recently (the name of this option was changed in main), and it looks like you don't have the newest changes applied. Can you rebase to make sure we don't lose the recent edits?

Reply:
Rebased